index.html

```diff
@@ -1770,6 +1848,28 @@
             default presentation.
           </div>
         </section>
+        <section>
+          <h4>
+            Receiving user agent
+          </h4>
+          <p>
+            The following partial interface MUST be implemented by a
+            <a>receiving user agent</a>:
+          </p>
+          <pre>
```

Comment:
Missing `class="idl"`.
Hmm, trying locally (with an empty interface definition), I see that ReSpec is getting really confused about references to the Presentation interface :(